package/boot/uboot-layerscape-armv7/patches/001-increse-CONFIG_SYS_BOOTM_LEN.patch

```diff
@@ -0,0 +1,28 @@
+From c268f9cdc3cb7bceb323a77a7f857d546a5bd30d Mon Sep 17 00:00:00 2001
+From: Xiaoliang Yang <xiaoliang.yang@nxp.com>
+Date: Tue, 11 Apr 2017 18:03:14 +0800
+Subject: [PATCH] increse CONFIG_SYS_BOOTM_LEN
+
+Set CONFIG_SYS_BOOTM_LEN to 10M
```

Comment:
An explanation of _why_ rather than just a rewording of the diff would be nicer IMO

Reply:
thank you for your review. I have changed the description, rewrite it to "increse CONFIG_SYS_BOOTM_LEN to 10M" and  deleted "Set CONFIG_SYS_BOOTM_LEN to 10M"